odata-json-format/15 Delta Payload.md

```diff
@@ -478,17 +492,152 @@ inline as a delta
     },
     {
       "@context": "#Customers/$link",
+      "@Org.OData.Core.V1.ContentID": "5",
       "source": "Customers('ANATR')",
       "relationship":" Orders",
       "target": "Orders(10643)"
     },
     {
       "@context": "#Customers/$deletedLink",
+      "@Org.OData.Core.V1.ContentID": "6",
       "source": "Customers('DUMON')",
       "relationship": "Orders",
       "target": "Orders(10311)"
     }
   ]
 }
 ```
+
+Assuming all changes can be applied without errors, the response would be
+```
+HTTP/1.1 204 No Content
+Preference-Applied: return=minimal, continue-on-error
+
+
+
+```
+
+Assuming some or all changes cannot be applied, the overall request is still deemed successful due to the `continue-on-error` preference, and the response details what went wrong
+  1. Add customer 'EASTC' - failed
+  2. Change `ContactName` of customer 'AROUT' - failed
+  3. Delete customer 'ANTON' - failed
+  4. Change customer 'ALFKI':
+     1. Create order 11011 - succeeded, not mentioned in response
+     2. Add link to existing order 10692 - succeeded, not mentioned in response
+     3. Change `RequiredDate` of related order 10835 - failed
+     4. Delete link to order 10643 - succeeded, not mentioned in response
+  5. Add link between customer 'ANATR' and order 10643 - failed without further info
+  6. Delete link between customer 'DUMON' and order 10311 - failed without further info
+```json
+HTTP/1.1 200 OK
+Content-Type: application/json
+Content-Length: ###
+Preference-Applied: return=minimal, continue-on-error
+
+{
+  "@context": "#$delta",
+  "value": [
+    {
+      "@Org.OData.Core.V1.ContentID": "1",
+      "CustomerID": "EASTC",
+      "@removed": {},
```

Comment:
Does `@removed` mean "not created" in this case?

Reply:
Seems so. We decided to "invert" the response: inserts become deletes, deletes become inserts, updates remain updates.